src/fragmentarium/domain/text.ts

```diff
@@ -140,16 +140,23 @@ export type Token =
   | Enclosure
   | Tabulation
 
+export type LineContainer = Line | RulingDollarLine
```

Comment:
This is not a container. It also seems redundant since `RulingDollarLine` is a `Line`.

How about renaming the `Line` below and having this like the tokens above `Line = RulingDollarline | ...`?